kernel/scheduler.c

```diff
@@ -1077,19 +1063,19 @@ static void task_create_real(struct task *new, struct vfs_info *vfsi, void *arg,
     new->tb.tracer = NULL;
     new->tb.timer_id = -1;
 
-    if ((new->tb.flags & TASK_FLAG_VFORK) != 0) {
+    if ((new->tb.flags &TASK_FLAG_VFORK) != 0) {
```

Comment:
I liked the spaces around the binary operator '&'. Apparently clang-format cannot tell this apart from the dereference '&' ?